src/Tests/Microsoft.NET.TestFramework/TestCommandLine.cs

```diff
@@ -29,10 +29,10 @@ public class TestCommandLine
 
         public string SdkVersion { get; private set; }
 
-        public string MSBuildSDKsPath { get; private set; }
-
         public string TestExecutionDirectory { get; set; }
 
+        public string MsbuildAdditionalSdkRsolverFolder { get; set; }
```

Comment:
Typo (here and elsewhere, including command line help)

```suggestion
        public string MsbuildAdditionalSdkResolverFolder { get; set; }
```